org.mwc.cmap.xyplot/src/org/mwc/cmap/xyplot/views/XYPlotView.java

```diff
@@ -1611,10 +1611,11 @@ public void run()
 			{
 				if (_plotControl != null && !_plotControl.isDisposed())
 				{
-					_plotControl.setSize(0, 0);
-					_plotControl.getParent().layout(true, true);
-					_plotControl.redraw();
-					_plotControl.update();
+					JFreeChart c = _plotControl.getChart();
+					if (c != null) 
+					{
+						c.setNotify(true);
+					}
 				}

```

Comment:
Here's Peco's description of this change:
    The old code was my trick to refresh _plotControl. Unfortunately, it doesn't 
    refresh time stepper :( . The new code is copied from JFreeChart's swt 
    chart composite (not sure exactly where I found it, but I already used it in a 
    mouse event). It refreshes complete chart properly.